Source/Atomic/UI/UI.cpp

```diff
@@ -535,11 +539,52 @@ void UI::HandleUpdate(StringHash eventType, VariantMap& eventData)
         exitRequested_ = false;
         return;
     }
+    
+    mouseStayedTime += eventData[Update::P_TIMESTEP].GetFloat();
+
+    if (mouseStayedTime >= 0.5f)
+    {
+        UIWidget* hoveredWidget = GetHoveredWidget();
+        if (hoveredWidget && !tooltip_ && (hoveredWidget->GetShorten() || hoveredWidget->GetTooltip().Length() > 0))
+        {
+            tooltip_ = new UIPopupWindow(context_, hoveredWidget, "tooltip");
```

Comment:
TBPopupWindow is designed to be attached to the `target` widget, and it's hard to deattach it from it an stick with another widget without recreating, so it's simplier to close the window and create a new one to the another widget(realocating doesn't consume a lot of resources and time, I've checked :smile: )
I can do it with realocating, but it needs much more TurboBadger code modification or creating a special widget for that(TBFloatingWindow?)